### DISCUSSION TITLE
Parser for Bun heap snapshot?

### DISCUSSION THREAD

**Discussion by vjpr** [oven-sh/bun#3804] [Q&A]
> https://github.com/oven-sh/bun/discussions/109: Bun.generateHeapSnapshot() returns a heap snapshot in a format I'm not entirely sure how to visualize yet

For V8 there is: https://github.dev/adriancooney/puppeteer-heap-snapshot

Does a package already exist for JSC's format?

**Comment by Rein1n** [comment]
bump, generated heap files w/ bun but I can't load them into chrome dev tools. Does a visualizer exist for this yet?

**Reply by jaredly** [reply]
you can load the JSC heap snapshot into safari https://bun.com/docs/project/benchmarking#view-example-statistics
